### PR TITLE
#5 simulate wind and power generation

### PR DESCRIPTION
**Pull request by 97gushan**
Adding issue #5, simulating wind and power generation

### CODE REVIEW

**Comment by 97gushan** [comment]
New power calculation function is based on the power output profile graph in: [Wind Statistics](http://www.wind-power-program.com/wind_statistics.htm)

Implementation gives a distribution displayed in the diagram:
PowerDistribution

File: Simulator/routes/api.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var express = require('express');
 var router = express.Router();
 
-var Simulator = require("../simulator");
+var Simulator = require("../simulation/simulator");
```
Comment (PandaFood) [inline Simulator/routes/api.js]: Avoid var in ES6

File: Simulator/simulation/simulator.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const WindTurbine = require('./windturbine');
+
```
Comment (PandaFood) [inline Simulator/simulation/simulator.js:2]: Is this beautified?
Lots of unnecessary  empty space

**Review by PandaFood — CHANGES_REQUESTED** [review]
Should testSigmoid.csv really be included?

**Review by PandaFood — APPROVED** [review]

**Pull request merged**